application/common/config/main.php

```diff
@@ -1,123 +1,32 @@
 <?php
 
 use Sil\PhpEnv\Env;
-use Sil\Log\EmailTarget;
 
-/*
- * Get config settings from ENV vars or set defaults
- */
-$mysqlHost = Env::get('MYSQL_HOST');
-$mysqlDatabase = Env::get('MYSQL_DATABASE');
-$mysqlUser = Env::get('MYSQL_USER');
-$mysqlPassword = Env::get('MYSQL_PASSWORD');
-$mailerUseFiles = Env::get('MAILER_USEFILES', false);
-$mailerHost = Env::get('MAILER_HOST');
-$mailerUsername = Env::get('MAILER_USERNAME');
-$mailerPassword = Env::get('MAILER_PASSWORD');
-$alertsEmail = Env::get('ALERTS_EMAIL');
-$alertsEmailEnabled = Env::get('ALERTS_EMAIL_ENABLED');
-$fromEmail = Env::get('FROM_EMAIL');
-$appEnv = Env::get('APP_ENV');
-$uiCorsOrigin = Env::get('UI_CORS_ORIGIN');
+$mysqlHost     = getRequiredEnv('MYSQL_HOST');
+$mysqlDatabase = getRequiredEnv('MYSQL_DATABASE');
+$mysqlUser     = getRequiredEnv('MYSQL_USER');
+$mysqlPassword = getRequiredEnv('MYSQL_PASSWORD');
+
+function getRequiredEnv($name)
```

Comment:
I had the same thought, I'd love to see it there if you think it's a common need.